api/CSSRule.json

```
@@ -67,7 +67,8 @@
               "version_added": "4"
             },
             "ie": {
-              "version_added": null
+              "version_added": null,
+              "notes": "Does not return the original verbatim rule text. E.g. 'border-width:1;border-style:solid;border-color:white;' will be returned as 'border: 1px solid white;'"
```

Suggesting some edits here for clarity and style:

```suggestion
              "notes": "The property normalizes rules and does not return the original verbatim rule text. For example, <code>border-width: 1px; border-style:solid; border-color:white;</code> returns <code>border: 1px solid white;</code>."
```